src/layer/ImageOverlay.js

```diff
@@ -165,6 +169,7 @@ export var ImageOverlay = Layer.extend({
 		img.onmousemove = Util.falseFn;
 
 		img.onload = Util.bind(this.fire, this, 'load');
+		img.onerror = Util.bind(this._overlayOnError, this, 'error');

```

Comment:
It looks like `_overlayOnError` takes no arguments so I think that `'error'` string will just disappear into the aether! https://github.com/Leaflet/Leaflet/blob/61ff641951fa64ba4730f71526988d99598681c8/src/layer/ImageOverlay.js#L231